src/main/webapp/WEB-INF/tags/enableJS.tag

```diff
@@ -0,0 +1,168 @@
+                <h3>MAC</h3>
+                <ol>
+                    <li>Select the Safari menu item from the Apple/System bar at
+                        the top of the screen.</li>
+                    <li>From the drop-down menu, select Preferences.</li>
+                    <li>Select the Content icon/tab at the top of the Preferences
+                        window.</li>
+                    <li>Then, check the Enable JavaScript checkbox.</li>
+                    <li>Click OK to save your changes and close the Preferences
+                        window.</li>
+                    <li>Finally, refresh your browser.</li>
+                </ol>
+            </div>
+
+            <div id=chrome>
+                <h1>Chrome</h1>
+                Please complete the following instructions to activate and enable
+                JavaScript in Chrome.
+
+                <h3>PC</h3>
+                <ol>
+                    <li>Select Customize and control Google Chrome (wrench Icon)
+                        to the right of the address bar.</li>
+                    <li>From the drop-down menu, select Options Select the Under
+                        the Hood tab at the top of the window.</li>
+                    <li>Under the Privacy heading, select the Content settings
+                        button.</li>
+                    <li>On the left, under the features heading, select
+                        JavaScript.</li>
+                    <li>Select the Allow all sites to run JavaScript radio button.</li>
+                    <li>Finally, close both preference windows, and refresh the
+                        browser.</li>
+                </ol>
+                <h3>MAC</h3>
+                <ol>
+                    <li>Select the Chrome menu item from the Apple/System bar at
+                        the top of the screen.</li>
+                    <li>From the drop-down menu, select Preferences...</li>
+                    <li>Select the Under the Hood tab at the top of the window.</li>
+                    <li>Under the Privacy heading, select the Content settings
+                        button.</li>
+                    <li>On the left, under the features heading, select
+                        JavaScript.</li>
+                    <li>Select the Allow all sites to run JavaScript radio button.</li>
+                    <li>Finally, close both preference windows, and refresh the
+                        browser.</li>
+                </ol>
+            </div>
+        </div>
+        <p>
+            As taken from:
+        </p>
+        <ol>
+            <li><a href="http://activatejavascript.org/en/instructions/">http://activatejavascript.org/en/instructions/</a></li>
+            <li><a href="http://support.mozilla.org/en-US/kb/javascript-settings-for-interactive-web-pages">
+                         http://support.mozilla.org/en-US/kb/javascript-settings-for-interactive-web-page</a></li>
+        </ol>
+    </div>
+</noscript>
```

Comment:
I believe a jsp include might work better for this in terms of performance.
Tag appears to be mainly for more dynamic pages and this is completely static.
I actually wanted it in the WEB-INF folder because of a relative linking problem ("../enableJS.jsp" refers to a different file while editing and during server running, requiring an additional enableJS.jsp in that folder for it to not report it as an error) and figured I should use .tag for consistency.

I'm going off for now, but [here's more information](http://stackoverflow.com/a/14763794).